apps/web/pages/availability/[schedule].tsx

```diff
@@ -80,7 +80,7 @@ const DateOverride = ({ workingHours }: { workingHours: WorkingHours[] }) => {
         <DateOverrideInputDialog
           workingHours={workingHours}
           excludedDates={excludedDates}
-          onChange={(ranges) => append({ ranges })}
+          onChange={(ranges) => ranges.forEach((range) => append({ ranges: [range] }))}
```

Comment:
We're now accepting an array of ranges for accommodating multiple date overrides